src/components/converter/assets/dai.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Should we remove these two svgs now?

Reply:
Reverting the `tokenIconUrl()` changes. They weren't being used, and they make it harder to show the icon on Rinkeby.